### DISCUSSION TITLE
Menubar and MegaMenu, autoDisplay to show/hide when hovering with the mouse?

### DISCUSSION THREAD

**Discussion by rubencl** [primefaces/community#1186] [PrimeVue]
I saw in some of the versions of Prime for other FWs there are autoDisplay and autoHide options for these components, but they don't exist in PrimeVue, where the behavior is always to show the menus when you click with the mouse, and you cannot change it for mouse hover.

I tried to set the behavior by adding in my MegaMenu @mousehover="toggle" or @mousehover="show" but it doesn't work.

Any suggestion? Are you planning to add these options to Menubar and Megamenu? Thanks!

**Comment by rafacribas** [comment]
Would love a response for this too

**Comment by igorraphael** [comment]
I got awnser from discord prime:
"You can use this a wrapper for opening the Megamenu on hover and you'd probably just have to update the PT targets to make it work with the Menubar": https://stackblitz.com/edit/uqcswx-n8kfcr?file=src%2FHoverMegaMenu.vue

**Reply by Marius-Ar** [reply]
It would be great if it was natively supported as this wrapper only works for a single Primefaces-supported framework and kind of needed for Angular too 😄
